### ISSUE TITLE
Facebook Connect - Extended Profile Infomation

### ISSUE THREAD

**Issue by Valentinez** [boonex/dolphin.pro#578]
When I enabled it, I was getting an error, turns out they do not use 'bio' anymore, they use 'about'
so I edited the code, seemed to work, but it's not populating any of the fields except for the first name being in the Username slot, and your e-mail address.
After I set a password, and click Next, no information was grabbed from Facebook, it's all blank.

**Comment by AlexTr** [comment]
As I see "bio" field is equivalent to "about" field:
https://developers.facebook.com/docs/graph-api/reference/user

Also to be able to collect these additional info your Facebook app must be manually approved for the same set of fields.

**Comment by Valentinez** [comment]
Yes, that is what I explained in my OP. I replaced the instances where "bio" appeared in the code, and replaced it with "about", and now I don't have an Error, but it still not grabbing the info.

Also, I'm the "developer" for the App, so I have the permissions required to obtain from the fields.
It is not grabbing the information however. It only grabs the first name, and e-mail address. No birthday, about, or anything else.

I'm sorry my first post wasn't clear: something is wrong in the code, and it needs to be fixed, but I'm not too good with code, replace some key words, sure, but I can't totally dissect code. I am however familiar with that fact that if I see one thing out of place ie. bio instead of about, there may be something else in the Facebook API that was changed that is preventing Dolphin from grabbing the information correctly, and therefore have made a guess that this code need to be updated to support the current API.
